frontend/src/components/mine/Permit/MinePermitInfo.js

```diff
@@ -60,44 +61,19 @@ const columns = [
 ];
 
 const childColumns = [
+  { title: "", dataIndex: "key", key: "Key" },
+  { title: "Received Date", dataIndex: "ReceivedDate", key: "ReceivedDate" },
+  { title: "Issue Date", dataIndex: "IssueDate", key: "IssueDate" },
```

Comment:
the render line is important for responsive views,

```
render: (text) => <div title="Permit No.">{text}</div>
```

CSS uses the title attribute to add a label for the card. 
Please add these back in.

Reply:
(note that the responsive nested table still needs some work, but it is being addressed in the next sprint)

Reply:
Good to know, thanks! I will add a render line. 👍

Reply:
Thank you!

Reply:
Added these back in. FYI I didn't intentionally remove them, it was merge side effect :)